src/modules/mints/MinterTable.tsx

```diff
@@ -0,0 +1,47 @@
+import moment from "moment";
+import Link from "next/link";
+import { FunctionComponent } from "react";
+import { MintEntities } from "../../types/entities";
+import { shortenHash } from "../../utils/sortenHash";
+
+type MinterTableProps = {
+	mintEntities: MintEntities[];
+};
+
+const MinterTable: FunctionComponent<MinterTableProps> = ({ mintEntities }) => {
+	return (
+		<table className="w-full border-collapse border border-slate-500">
+			<tr>
+				<th className="border border-slate-600">txHash</th>
```

Comment:
it's redundant, how about you create your own `TH` component? 
or event your own table component ? 

but for now it think it's okay